app/email_utils.py

```diff
@@ -1513,3 +1521,38 @@ def get_verp_info_from_email(email: str) -> Optional[Tuple[VerpType, int]]:
     if data[2] > time.time() + VERP_MESSAGE_LIFETIME:
         return None
     return VerpType(data[0]), data[1]
+
+
+# This method processes the email address, checks if it's a signed verp email generated by us to receive bounces
```

Comment:
Nits: can we move these comments to the method doc?

Reply:
Done